docker/lite/Dockerfile.alpine

```diff
@@ -51,8 +51,6 @@ ENV MYSQL_FLAVOR MariaDB103
 
 # Copy artifacts from builder layer.
 COPY --from=builder --chown=vitess:vitess /vt/install /vt
-COPY --from=builder --chown=vitess:vitess /vt/src/vitess.io/vitess/web/orchestrator /vt/web/orchestrator
-COPY --from=builder --chown=vitess:vitess /vt/src/vitess.io/vitess/web/vtorc /vt/web/vtorc
```

Comment:
Since `/web/vtorc` and `/web/orchestrator` have been removed, we don't need to copy them in the `lite` images.